session/bootstrap.go

```diff
@@ -240,6 +240,7 @@ const (
 		update_time timestamp(3) NOT NULL,
 		charset text NOT NULL,
 		collation text NOT NULL,
+		create_way text NOT NULL,
```

Comment:
We'd better use `alter table mysql.bind_info add column ...` and add a new version for bootstrap to avoid upgrade compatibility problems.

Reply:
BTW, a default value `unkown` for `create_way` is also necessary. Because the lower version of tidb-server doesn't have this column.

Reply:
done now, PTAL @lzmhhh123